compiler/driver.ml

```diff
@@ -1204,6 +1204,11 @@ let main () =
     Message.Content.emit content Error;
     if Global.options.debug then Printexc.print_raw_backtrace stderr bt;
     exit Cmd.Exit.some_error
```

Comment:
I might like a debug message telling me that there were pending delayed errors when a non-delayed fatal error cut them short :)